articles/cosmos-db/concepts-limits.md

```diff
@@ -152,9 +152,11 @@ Here's a list of limits per account.
 
 | Resource | Limit |
 | --- | --- |
-| Maximum number of databases and containers per account  | 100 |
+| Maximum number of databases and containers per account  | 100 ¹ |
 | Maximum number of regions | 1 (Any Azure region) |
 
+¹ You can increase this limit by creating an [Azure Support request](create-support-request-quota-increase.md) up to 500 max.
```

Comment:
```suggestion
¹ To increase this limit, create an [Azure support request](create-support-request-quota-increase.md).
```

Minor edits. To increase the longevity of this article, it's best to not publish the support max as this number may increase in the future.

Reply:
@seesharprun  Agreed! Have removed the limit, but maintained the wording as per the rest of the document - keeping the footer note consistent e.g. https://learn.microsoft.com/en-us/azure/cosmos-db/concepts-limits#:~:text=%C2%B9%20You%20can%20increase%20Maximum%20RUs%20per%20container%20or%20database%20by%20filing%20an%20Azure%20support%20ticket.